docs/getting_started/basics/nesting_workflows.md

```diff
@@ -1,16 +1,22 @@
 # Nesting Workflows
 
-In Shift any workflow can be added and executed as an operator.
+In Shift, any workflow can be embedded into a node and executed within another workflow.
 
 ## Setting Workflow I/Os
+To create a reusable workflow, the input and output of the workflow must be specified. This is done through `Input` and `Output` nodes. The plugs created in these nodes define which inputs are expected by the workflow and which are the outputs that will be returned.
 
-First of all we need to tell Shift which are the *Inputs* and *Outputs* of the workflow in question. To do that we make use of special operators called `Input` and `Output`. Then we need to create plugs for these operators corresponding to what we expect to be requested by the workflow and what we want to be returned.
+![Workflow I/Os](../../images/workflow_ios.png)
 
-![Workflow I/Os](../images/workflow_ios.gif)
+The plugs from these nodes will act as data pathways, linking information set externally in the workflow to the current workflow and viceversa.
 
-## Workflow Operator
+## Workflow Node
 
-The `Workflow` operator allows to embed the execution of a workflow file inside another workflow as an operator.
-When setting the file plug content to source a valid *.sft* file, the node will automatically update adding the input and output plugs corresponding to the workflow inputs and outputs.
+The `Workflow` node allows to embed the execution of a workflow inside another workflow. This node has a *file* plug, where the workflow to nest is to be indicated. When the file plug content is set to reference a valid workflow file (with a *.sft* extension), the node will automatically update by creating the input and output plugs corresponding to the sourced workflow inputs and outputs. Then, when the `Workflow` node gets executed, it will execute the referenced workflow. 
 
-![Nested Workflows](../images/nested_workflows.gif)
+![Nested Workflows](../../images/workflow_nesting.gif)
+
+To learn more about the `Workflow` node functionalities, please refer to the [`Workflow`](../../nodes/workflow.md) documentation.
+
+### Workflows on External Processes
+
+When using the `Workflow` node, the execution of the workflow happens on the same python process Shift is running in. However, its possible to define and execute a workflow in an external process. To do so, please refer to the [`Workflow Process`](../../nodes/workflow_process.md) documentation.
```

Comment:
```suggestion
When using the `Workflow` node, the execution of the workflow happens on the same python process Shift is running in. However, its possible to define and execute a workflow in an external process or with a different Python interpreter. To do so, please refer to the [`Workflow Process`](../../nodes/workflow_process.md) documentation.
```

Reply:
It's is not an "or" right? When launching a workflow with another python interpreter you are executing the workflow in an external process

Reply:
It's an external process handled by a Python interpreter which can be the same one as the current session of Shift or a different one. I think the important feature to talk about here is the fact that you can set a different interpreter, not that it runs in a subprocess. Also as the Python execution is single threaded, there is no gain in executing in a subprocess but the fact that you can change intepreter... as you still have to wait for the process to complete execution before being able to continue. I think we can probably rephrase the statement?

It is not an xor xD

Reply:
What do you think about:
"However, it's possible to define and execute a workflow in an external process running in a different Python interpreter."

Reply:
Done!